workflow-templates/test-package.yaml

```diff
@@ -39,7 +42,6 @@ jobs:
     needs: analyze
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: false
```

Comment:
Removing this means that as soon as any of the jobs in the matrix fails, all other jobs will be cancelled immediately. That's different from what I've seen on travis.

Reply:
Oh! So this makes sure this runs if the previous job fails. Got it.

Again, just for symmetry I'd add it above then

Reply:
fixed